CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [2.2.1] - 2022-03-10
+### Fixed
+- Fixed incorrect logic for execution mode calculation.
+- Set context.beacon=false for server side.
```

Comment:
For the second bullet point let's just say "Always use context.beacon=false for sendNotifications() calls"